src/components/core/base/VisualizationLoadingHOC.tsx

```diff
@@ -354,12 +354,32 @@ export function visualizationLoadingHOC<
                     return Promise.reject(new Error("Unknown projectId"));
                 }
 
-                const title = exportTitle ? escapeFileName(exportTitle) : "Untitled";
+                const {
+                    format,
+                    includeFilterContext,
+                    mergeHeaders,
+                    showFilters,
+                    title: customTitle,
+                } = exportConfig;
+
+                const title: string = escapeFileName(customTitle || exportTitle) || "Untitled";
+
+                const exportConfigRequest: IExportConfigRequest = {
+                    format,
+                    mergeHeaders,
+                    title,
+                };
+                if (includeFilterContext) {
```

Comment:
Please add definition of this change into changelog and mention priority of these two ways how to define filters for export

Reply:
removed the backward compatibility's code